addons/website/static/src/xml/website_form_editor.xml

```diff
@@ -5,17 +5,11 @@
     <t t-name="website.s_website_form_end_message">
         <div class="s_website_form_end_message d-none">
             <div class="oe_structure">
-                <section class="s_text_block pt64 pb64 o_colored_level o_cc o_cc2" data-snippet="s_text_block">
-                    <div class="container">
-                        <h2 class="text-center">
-                            <span class="fa fa-check-circle"/>
-                                Thank You For Your Feedback
-                        </h2>
-                        <p class="text-center">
-                            Our team will message you back as soon as possible.<br/>
-                            In the meantime we invite you to visit our <a href="/">website</a>.<br/>
-                        </p>
-                    </div>
+                <section class="s_text_block o_colored_level rounded px-5 text-center" data-snippet="s_text_block">
+                    <i class="fa fa-paper-plane fa-2x mb-3 rounded-circle text-bg-success" role="img" aria-label="Success" title="Success"/>
```

Comment:
Conflict aside (on the design PR), it seems all good to me.
Just here: I don't think a role and aria-label are needed. It's a purely decorative icon which doesn't bring information nor context, it could (and probably should) be set to `aria-hidden="true"`.

Reply:
I've replaced `aria-label="Success"` with `aria-hidden="true"`.
But for the `role`, we should keep it 😉 : https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA24

Reply:
Cool! I had doubts for the role, it's nice that you found the answer. :)

The 2 other icons in `website_form.xml` are also just decorative and could get the same treatment. :+1:

Reply:
Done ✅